src/WorkBC.Importers.Federal/Services/XmlImportService.cs

```diff
@@ -318,8 +318,8 @@ public async Task UpdateJobs()
                             continue;
                         }
 
-                        //get XML from URL if needed
-                        await GetXmlContent(federalJob.JobId, true, true);
+                        // get french and english job descriptions from Federal Job Board API
+                        await GetXmlContent(federalJob.JobId);
```

Comment:
The english and french descriptions are always requested, so remove the unused parameters